code/modules/antagonists/changeling/powers/transform.dm

```diff
@@ -113,6 +113,8 @@
 	var/icon/cached_flat_icon
 	/// HUD job icon of the ID
 	var/hud_icon
+	/// Gun permit of the ID (TRUE or FALSE, it either has one or it doesn't).
+	var/gun_permit // SKYRAT EDIT
```

Comment:
All things considered, this should also be moved to a modular folder, since it's something that doesn't require it to be non-modular.